pyface/tests/test_color.py

```diff
@@ -0,0 +1,234 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+from unittest import TestCase
+
+from traits.testing.unittest_tools import UnittestTools
```

Comment:
Nit: It is available from `traits.testing.api`, it would be preferred to import from `api`.